test-skripsi/test-prod.tsx

```diff
@@ -1,5 +1,12 @@
+class InventoryError extends Error {
+	constructor(message: string) {
+		super(message);
+		this.name = "InventoryError";
+	}
+}
+
 interface Product {
-	id: number;
+	readonly id: number;
```

Comment:
Making `id` readonly is a good practice to ensure the immutability of the identifier. Consider applying similar constraints to other critical properties.